creator-node/test/lib/blacklistManager.js

```diff
@@ -16,10 +16,6 @@ async function restartBlacklistManager(redis) {
     keyPattern: BlacklistManager.getRedisTrackIdToCIDsKey('*')
```

Comment:
should we put these tests back in to verify this functionality? https://github.com/AudiusProject/audius-protocol/pull/3250/files#diff-959a9d0b973c6d5c2a4879d1b0cb6c34a4c0d4eeb276f98fffdce49c3e040845R711